app/src/components/Register.tsx

```diff
@@ -145,6 +197,14 @@ class Register extends React.Component<RegisterProps, RegisterState> {
     this.setState(stateUpdate);
   };
 
+  getOptions = (): any => {
```

Comment:
Any?

Reply:
Ah yes I can't work out how to type a list of <option>, what type is <option>?